addons/sale_timesheet/report/timesheets_analysis_report.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+class TimesheetsAnalysisReport(models.Model):
+
+    _inherit = "timesheets.analysis.report"
+    _auto = False
```

Comment:
No need when you inherit to a same model. 🙂 
It is needed when you want to override the value set to this attribute for this report. 🙂 
```suggestion
```